### PR TITLE
1000 test file refactor

### PR DESCRIPTION
**Pull request by SuttonMashing**

### CODE REVIEW

File: spec/system/sponsor_additional_details_spec.rb [hunk 1]
```diff
@@ -59,16 +57,25 @@
 
     it "validates other identity documents field on any submission" do
```
Comment (neilkidd) [inline spec/system/sponsor_additional_details_spec.rb:58]: Not sure the "it" actually describes the test now?

File: spec/system/sponsor_additional_details_spec.rb [hunk 2]
```diff
@@ -59,16 +57,25 @@
...
       navigate_to_additional_details
-      choose("I don't have any of these")
+      uam_enter_sponsor_identity_documents("I don't have any of these")
+
+      expect(page).to have_content("Enter your date of birth")
+    end
+
+    it "does not allow for empty nationality to be selected" do
```
Comment (neilkidd) [inline spec/system/sponsor_additional_details_spec.rb:65]:
```suggestion
    it "enforces that nationality is mandatory" do
```

File: spec/system/sponsor_additional_details_spec.rb [hunk 3]
```diff
@@ -59,16 +57,25 @@
...
+    end
 
-      fill_in("unaccompanied-minor-no-identification-reason-field", with: "Hello")
+    it "shows an error box when no nationality is selected " do
+      navigate_to_nationality
 
       click_button("Continue")
 
-      expect(page).to have_content("Enter your date of birth")
+      expect(page).to have_content("There is a problem")
```
Comment (neilkidd) [inline spec/system/sponsor_additional_details_spec.rb:78]: I think this assertion can be moved into the above test as the arrange and act steps are the same.

File: spec/system/uam_sponsor_contact_details_spec.rb [hunk 4]
```diff
@@ -11,6 +11,27 @@
   let(:nonmatching_sponsor_email) { "notmatching@example.com" }
   let(:phone_page_content) { "Enter your UK mobile number" }
   let(:task_list_content) { "Apply for approval to provide a safe home for a child from Ukraine" }
+  let(:valid_phone_number) { "07777 888 999" }
+
+  describe "Sponsor fills out contact details" do
+    it "Answers are saved to the db" do
```
Comment (neilkidd) [inline spec/system/uam_sponsor_contact_details_spec.rb:17]:
```suggestion
    it "Answers are persisted and visible on returning to the page" do
```

File: spec/system/uam_sponsor_contact_details_spec.rb [hunk 5]
```diff
@@ -11,6 +11,27 @@
...
+      fill_in_email_and_continue
+
+      fill_in_phone_numbers_and_continue
+
+      expect(page).to have_content(task_list_content)
```
Comment (neilkidd) [inline spec/system/uam_sponsor_contact_details_spec.rb:22]:
```suggestion
      uam_enter_sponsor_contact_details
```

**Review by neilkidd — CHANGES_REQUESTED** [review]

File: spec/system/sponsor_additional_details_spec.rb (hunk 3 above)
Comment (SuttonMashing) [reply spec/system/sponsor_additional_details_spec.rb:70]: I agree, done it

**Review by neilkidd — APPROVED** [review]
🎉🎉🎉🎉

**Pull request merged**